packages/wallet/src/validator/rollup-state-validator.ts

```diff
@@ -177,33 +163,39 @@ export class DefaultRollupStateValidator implements RollupStateValidator {
       generatedPostRoot = await this.rollupMachine.getStateRoot()
     } catch (error) {
       if (isStateTransitionError(error)) {
-        log.info(
-          'The transaction did not pass the state machine, must be badly formed!  Returning fraud proof.'
+        log.error(
+          `The transaction did not pass the state machine, must be badly formed!  Returning fraud proof. Transition: ${JSON.stringify(
+            nextTransition
+          )}`
         )
         return {
           fraudPosition: this.currentPosition,
           fraudInputs: preppedFraudInputs,
           fraudTransition: nextTransition,
         }
       } else {
-        log.error(
+        logError(
           log,
-          'Transaction ingestion threw an error--but for a reason unrelated to the transition itself not passing the state machine. Uh oh!',
+          `Transaction ingestion threw an error--but for a reason unrelated to the transition itself not passing the state machine. Uh oh! Transition: ${JSON.stringify(
+            nextTransition
+          )}`,
           error
         )
         throw new LocalMachineError()
       }
     }
 
     if (generatedPostRoot.equals(transitionPostRoot)) {
-      log.info(
-        'Ingested valid transition and postRoot matched the aggregator claim.'
+      log.debug(
+        `Ingested valid transition and postRoot matched the aggregator claim.`
       )
       this.currentPosition.transitionIndex++
       return undefined
     } else {
-      log.info(
-        'Ingested valid transition and postRoot disagreed with the aggregator claim--returning fraud'
+      log.error(
+        `Ingested valid transition and postRoot disagreed with the aggregator claim--returning fraud. Transition: ${JSON.stringify(
+          nextTransition
+        )}. Expected State Root: ${hexBufToStr(generatedPostRoot)}`
```

Comment:
Great to see how you did logging here--I guess the rule of thumb to have is if you're logging something that happened, also log the relevant objects causing it to happen so you can immediately get insight when debugging.  Makes sense!